tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -35,39 +44,250 @@ class TestVirtualMachineLifecycle(ManageTest):
+                    c in "0123456789abcdefABCDEF" for c in candidate
+                ):
+                    md5sum_output = candidate
+                    break
+
+        if md5sum_output is None:
+            logger.warning(f"Could not parse md5sum from output:\n{child.before}")
+        else:
+            logger.info(f"Parsed md5sum checksum: {md5sum_output}")
+        return md5sum_output
+
+    def _wait_for_vmi_agent_connected(self, vm_name, namespace, timeout=600):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_wait_for_vmi_agent_connected`

Add return type annotation: `None`

(ANN202)